src/services/users/create-new-user/create-new-user.test.js

```diff
@@ -0,0 +1,66 @@
+const { expect } = require('chai')
+const { faker } = require('@faker-js/faker')
+const db = require('../../../db')
+const createNewUser = require('./create-new-user')
+
+const addUserToInvite = async () => {
+	const signUpEmail = faker.internet.email()
+	const { rows:inviteRows }=  await db.query(
+		`INSERT INTO user_invites 
+        ("email") VALUES ($1) RETURNING *`, [signUpEmail])
+        
+	return inviteRows[0]
+}
+
+const updateUserToActive = async (email) => {
```

Comment:
let's move this to the test/utils as well. I think the name should also focus on the invite. maybe `updateInviteStatus` to be a bit more generic and take in the email and status or something better